pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -128,10 +128,9 @@ func (cb *ClusterBuilder) buildSubsetCluster(opts buildClusterOpts, destRule *co
 		clusterType = cluster.Cluster_ORIGINAL_DST
 	}
 	if !(isPassthrough || clusterType == cluster.Cluster_EDS) {
-		if len(subset.Labels) != 0 {
-			lbEndpoints = cb.buildLocalityLbEndpoints(proxyNetworkView, service, opts.port.Port, []labels.Instance{subset.Labels})
-		} else {
-			lbEndpoints = cb.buildLocalityLbEndpoints(proxyNetworkView, service, opts.port.Port, nil)
```

Comment:
where did this go?

Reply:
lbEndpoints = cb.buildLocalityLbEndpoints(proxyNetworkView, service, opts.port.Port, []labels.Instance{subset.Labels})

It is replaced with this single call. Basically if else not needed here.

Reply:
I see, so `nil` and `[]labels.Instance` are equivilent. Makes sense. Let me double check the original issue I am thinking of, but looks fine